lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -287,6 +291,26 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
       ]
     );
   }
+
+  /// For browsers that report delta line instead of pixels such as FireFox
+  /// compute line height using the default font size.
+  ///
+  /// Use Firefox to test this code path.
+  double _computeDefaultScrollLineHeight() {
+    const double kFallbackFontHeight = 16.0;
+    final html.DivElement probe = html.DivElement();
+    probe.style
+        ..fontSize = 'initial'
+        ..display = 'none';
+    html.document.body!.append(probe);
```

Comment:
Remove the probe before returning from this function?

Reply:
Done.